docs/modules/ROOT/pages/how-to/organizations.adoc

```diff
@@ -0,0 +1,43 @@
+= Managing Organizations
+
+One of the basic features of the {product} Portal is to provide a centralized location for users to manage access to their projects across all {product} zones.
+
+== Concepts
+
+{product} allows namespaces and projects to belong to a particular {product} organization, as explained in the xref:user:how-to/manage-projects-and-namespaces.adoc[Manage Projects and Namespaces] documentation page. On the other hand, the https://control.vshn.net/[VSHN Control Panel] also provides customer management features. Finally, OpenShift also provides a notion of Groups. How are all of these concepts related?
+
+{product} Organizations are related to OpenShift Groups in the sense that there will be a group for each organization on each zone. This makes them independent and cross-zones; this means that a {product} Organization spans across {product} zones automatically, allowing users to grant or deny access to projects from a central location.
+
+In order to be able to assign users to organizations, users need to xref:how-to/find-username.adoc[know their username]. Users not assigned to an organization are reminded to join one as soon as they log in to the {product} Portal for the first time.
+
+image::how-to/organizations/welcome.png[]
```

Comment:
Please move this also to the find username page. Probably needs some contextual words (perspective of find username is that of a regular user, in contrast to this page that addresses organization owners.

Reply:
But this is specifically about organizations, I do not understand how this makes sense in the username page. That's why I simply linked to it.

Reply:
I depend on the angle you look at it, I guess. I was more considering the "looking up my namespace aspect" which is also part of that welcome dialog. But I see, that it also fits as it is now. Let us call it good enough for now. We always can improve (for instance, when we get some user feedback).